frontend/src/component/feature/FeatureView/FeatureOverview/NewFeatureOverviewEnvironment/NewFeatureOverviewEnvironment.tsx

```diff
@@ -52,62 +52,75 @@ const StyledHeaderTitle = styled('span')(({ theme }) => ({
 }));
 
 interface INewFeatureOverviewEnvironmentProps {
-    environmentId: string;
+    hiddenEnvironments: string[];
```

Comment:
This seems like a cool change. how are we going from one env to multiple? I couldn't work it out from the description. Presumably this gives us some new capabilities? what component is this (do you have a screenshot)?

Reply:
Entire right side (list of environments)

Reply:
I ... don't quite understand what you're referring to 😅 But that's alright